CHANGELOG.md

```diff
@@ -2,3 +2,26 @@
 ## 0.1.0 (Sept 12, 2016)
 
 * Initial public release
+
+## 0.2.0 (Sept 22, 2016)
+
+
+### BREAKING CHANGES
+#### `kyt test` now uses [Jest](https://github.com/NYTimes/kyt/commit/55d51626405cb2be3046cc78e38d285e8116d1a3)
+
+See the new [testing docs](/docs/commands.md#test).
+Read more about [Jest](https://facebook.github.io/jest/)
+
+If you've already written tests in Ava check out these code mods from [mikenikles](https://github.com/mikenikles/jscodeshift-ava-to-jest) and [dcousineau](https://gist.github.com/dcousineau/0170c4cf30bcc83ebf1a39b4636ae7d6) as a resource for upgrading.
+
+#### support of other ESLint and StyleLint file types
+.eslintrc --> .eslintrc.json
+.stylelintrc --> .stylelintrc.json
+
+kyt adds these linter files to your project on setup. These files now reference base linter files in kyt to allow for easier upgrades. As always you can use these files to create your own overrides.
+
+#### kyt's global variables are now in a [KYT global object](https://github.com/NYTimes/kyt/commit/4d934378c9271b9b4115e42619d78fcc09ea2d7c) rather than process.env
```

Comment:
Can we point to the docs: https://github.com/NYTimes/kyt/blob/master/docs/conventions.md#environment-variables